deploy/role.yaml

```diff
@@ -1,81 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I guess we can also remove `/deploy/role_binding.yaml` too now?

Reply:
sure 6677503